lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -130,6 +132,23 @@ def docs
 
     private
 
+    def capture3(command)
+      env_vars = command.include?('/opt/puppetlabs') ? clean_env_vars : ::ENV
```

Comment:
Limiting the set of env vars breaks outside of the Puppetlabs environment. Keeping them as is seems to make sense in an environment where Puppet is available in the same Ruby context as kafo_parsers. This is real world only in a testing environment.

Reply:
This probably breaks because command is an array. Probably safer to use `self.class.puppet_bin`, especially when you make the command `run_puppet`.